bigquery-connector-common/src/main/java/com/google/cloud/bigquery/connector/common/IdentityTokenSupplier.java

```diff
@@ -41,6 +42,24 @@ public IdentityTokenSupplier(String audience) {
     lastFetchTime = 0;
   }
 
+  @Override
+  public int hashCode() {
+    return Objects.hashCode(cachedToken, lastFetchTime);
```

Comment:
Should audience be a part of the hashcode, equals?

Reply:
The cachedToken depends on the audience so it would be redundant.